.github/workflows/ci.yml

```diff
@@ -2,18 +2,30 @@ name: CI
 
 on:
   push:
-    paths-ignore:
-      - 'docs/**'
-      - '*.md'
+    branches: [ master ]
```

Comment:
please restore the paths ignore and disable the branch check

Reply:
this is an old diff